Shared/DatabaseDefintion/EntityModel/Repositories/CartRepository.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading;
+using System.Threading.Tasks;
+using DatabaseDefinition.EntityModel.Repositories.Interfaces;
+using DatabaseDefintion.EntityModel.Database;
+using Microsoft.EntityFrameworkCore;
+using TM = TransferModel;
+
+namespace DatabaseDefinition.EntityModel.Repositories;
+
+public class CartRepository : ICartRepository
+{
+    private readonly AppProjectDbContext dbContext;
+
+    public CartRepository(AppProjectDbContext dbContext)
+    {
+        this.dbContext = dbContext ?? throw new ArgumentNullException(nameof(dbContext));
+    }
+
+    public async Task AddItemToCartAsync(string userId, long itemId, CancellationToken cancellationToken)
+    {
+        _ = await dbContext.Items.Select(ItemMappings.MapItem)
```

Comment:
Gedanken dazu, wir haben das ja immer wieder. Sollten wir in den Entsprechenden Mappings Klassen das einmal statisch definieren? Also einmal für jede Tabelle wo wir das immer überprüfen.
Oder wir machen ein RepositoryHelper.
Oder wir benennen die Mappings in [name]Helper um. (bin ich glaube am meisten Fan von)